runtime/cfdumper/main.c

```diff
@@ -1437,12 +1437,22 @@ static void printDisassembledMethod(J9CfrClassFile* classfile, J9CfrMethod* meth
 			NEXT_U8_ENDIAN(bigEndian, bc, bcIndex);
 			if(wide)
 			{
-				j9tty_printf( PORTLIB, "%s ", sunJavaBCNames[bc]);
+				j9tty_printf( PORTLIB, "%s", sunJavaBCNames[bc]);
```

Comment:
Why the spacing change (in the string)?

Reply:
The spacing change is so that the bytecode can be appended to the bytecode name, in the case of JBunimplemented. This allows for easier debugging.

Example of new unimplemented output (prior to handling `204` as `JBwithfield`):
```
    2 JBunimplmented_204 26 FlattenedLine2D.en QPoint2D;
```

Space was removed to avoid confusion between the bytecode (204) and other data (26)